docs/_tutorials/zero.md

```diff
@@ -270,8 +270,8 @@ You can use this method to save ZeRO-2 weights as well.
 If you'd like to get the fp32 weights, we supply a special script that can do offline consolidation. It requires no configuration files or GPUs. Here is an example of its usage:
 
 ``` bash
-$ cd /path/to/checkpoints_dir
-$ ./zero_to_fp32.py global_step1 pytorch_model.bin
+$ cd /path/to/checkpoint_dir
+$ ./zero_to_fp32.py . pytorch_model.bin

```

Comment:
Is global_step1 still correct in the next line?

Reply:
I changed it to match the `load_checkpoint` algorithm, where only the top path needs to be provided and it then automatically picks the latest checkpoint unless tag is provided (and the script itself doesn't have an option to allow for the tag, but it's trivial to add as the main API supports it).

before that the user had to explicitly pass the checkpoint name.